inox2d/src/math/interp.rs

```diff
@@ -1,28 +1,6 @@
 use glam::Vec2;
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub enum InterpolateMode {
-	/// Round to nearest
-	Nearest,
-	/// Linear interpolation
-	Linear,
-	// there's more but I'm not adding them for now.
-}
```

Comment:
Like `TransformOffset`, there's no reason to move `InterpolateMode` in `crate::node::data`. It was fine as it is here.